baybe/kernels/base.py

```diff
@@ -19,11 +19,19 @@
 if TYPE_CHECKING:
     import torch
 
+    from baybe.surrogates.gaussian_process.kernel_factory import PlainKernelFactory
+
 
 @define(frozen=True)
 class Kernel(ABC, SerialMixin):
     """Abstract base class for all kernels."""
 
+    def to_factory(self) -> PlainKernelFactory:
+        """Wrap the kernel in a :class:`baybe.surrogates.gaussian_process.kernel_factory.PlainKernelFactory`."""  # noqa: E501
+        from baybe.surrogates.gaussian_process.kernel_factory import PlainKernelFactory
```

Comment:
y u double import?

Reply:
Upper is required for type check, lower is to avoid cyclic imports: `Kernel.to_factory` requires `PlainKernelFactory`, but the file in which the latter resides currently requires `Kernel` in the converter. Could be alternatively resolved if I moved the converter utility to the kernel file. However, I somehow thought it rather belongs to the factory? Let me know if you prefer a different solution